app/controllers/universities_controller.rb

```diff
@@ -0,0 +1,56 @@
+class UniversitiesController < ApplicationController
+
+  authorize_resource
+
+  def index
+    @universities = University.all
+    @universities_page = @universities.page(params[:page])
+    respond_to do |format|
+      format.html
+      format.csv
+      format.xls  #{ send_data @universities.to_csv(col_sep: "\t") }
+    end
+  end
+
+  def show
+    @university = University.find(params[:id])
+  end
+
+  def new
+    @university = University.new
+  end
+
+  def edit
+    @university = University.find(params[:id])
+  end
+
+  def create
+    @university = University.new(resource_params)
+    if @university.save
+      redirect_to universities_path
+    else
+      render 'new'
+    end
+  end
+
+  def update
+    @university = University.find(params[:id])
+    if @university.update(resource_params)
+      redirect_to universities_path
+    else
+      render 'edit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.